app/javascript/Pages/Connections.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Layout from "./Layout";
+import { Head, Link } from "@inertiajs/inertia-react";
+import "../stylesheets/body.css";
+
+export default function Welcome(props) {
+  const token = document.querySelector(
+    "[name=csrf-token]"
+  ).content;
+  const headers = { "X-CSRF-Token": token };
+
+  const Connections_list = (props) => {
```

Comment:
**issue (blocking)**: you need to be consistent in the names you use. functions start with lower case and use camel case (e.g.: connectionsList), components start with upper case and also use camel case (e.g.: ConnectionsList)